metricflow-semantics/metricflow_semantics/test_helpers/semantic_manifest_yamls/simple_manifest/metrics.yaml

```diff
@@ -805,3 +805,36 @@ metric:
       name: listings
   filter: "{{ Metric('views', ['listing']) }} > 10"
   time_granularity: week
+---
+# note that in both 1a and 1b "shared_alias" refer to "bookinks", whereas
+# in 2 it refers to "instant_bookings", hence the naming difference
+metric:
+  name: derived_sharedalias_1a
```

Comment:
can we add an underscore between `shared` and `alias` in these metric names? just to improve readability